index.html

```diff
@@ -423,8 +473,13 @@
         acquiring permission or determining the permission status.
       </p>
       <p>
-        When a permission is revoked, all <a>push subscriptions</a> created with that permission
-        MUST be <a>deactivated</a>.
+        When a permission is revoked, the <a>user agent</a> MAY
```

Comment:
What about if permission is granted again, after being revoked? @beverloo, I think you mentioned Chrome won't fire a `pushsubscriptionchange` event in that case, right?